src/test/resources/cucumber/features/Modernisation.jurorResponses/JM-6064.feature

```diff
@@ -1,154 +1,392 @@
+    And I see "Daily expenses" on the page
+    And I press change default expenses button
 
-        #enter more than daily limit
-        #check that financial loss daily limit screen appears
+    And I set "Loss of earnings or benefits per day" to "100"
+    And I set "Hour" to "3"
+    And I set "Minute" to "15"
+    And I set "Miles travelled by car, motorcycle or bicycle per day" to "3"
+    And I set the radio button to "Yes"
+    And I set "Smartcard Number" to "123456789"
+    And I check the "Tick to overwrite any existing draft expenses with these amounts for this juror." checkbox
+    And I press the "Save default expenses" button
 
+    And I see "Daily expenses" on the page
+    And I see "3 hours 15 minutes" in the same row as "Travel time"
+    And I see "£100.00" in the same row as "Loss of earnings or benefits per day"
+    And I see "3 miles" in the same row as "Miles travelled"
+    And I see "123456789" in the same row as "Smartcard number"
 
+    And I press the "Apps" button
+    And I click on the "Juror management" link
+    And I click on the "Record attendance" link
+    When I press the "Confirm attendance" button
+    And I set "Hour" to "5"
+    And I set "Minute" to "30"
+    And I set the radio button to "pm"
+    And I press the "Continue" button
+    And I press the "Confirm attendance list is correct" button
 
-        
+    And the user searches for juror record "<juror_number>" from the global search bar
+    And I click on the "Expenses" link
+    And I click on the "View all expenses" link
+    And I click on "View expenses" in the same row as "<juror_number>"
+    And I see the unpaid expenses table
 
+        #edit draft expenses
+    And I click on my draft expense link with todays date
+    And I set the radio button to "Half day"
+    And I see "Daily expenses" on the page
+    And I see "Time" on the page
+    And I see "Financial loss" on the page
+    And I see "Travel" on the page
+    And I see "Payment method" on the page
+    And I see "Food and drink" on the page
+    And I see "Apply to all days" on the page
+    And I see "Totals" on the page
 
+        #change time/error checks
+    And I set "Hour" to "67"
+    And I set "Minute" to "98"
 
+        #change financial loss/error checks
+    And I set "Loss of earnings or benefits" to "123456!!"
+    And I set "Extra care costs" to "123456!!"
+    And I set "Other costs" to "123456!!"
+    And I set "Description of other costs" to "Testingothercosts12!)"
+
+        #change Travel/error checks
+    And I check the "Car" checkbox
+    And I see "Number of other jurors taken as passengers" on the page
+    And I set "Number of other jurors taken as passengers" to "123456!!"
+    And I check the "Motorcycle" checkbox
+    And I check the "Bicycle" checkbox
+    And I set "Parking" to "123456!!"
+    And I set "Public transport" to "123456!!"
+    And I set "Taxi" to "123456!!"
+
+        #change food and drink/error checks
```

Comment:
Do we need to do these error checks on all tests? I think checking for them on a single test should be sufficient.